lib/ts/recipe/multifactorauth/recipeImplementation.ts

```diff
@@ -437,55 +441,77 @@ export default function getRecipeInterface(
             const sessionUser = await getUser(session.getUserId(), userContext);
 
             if (sessionUser === undefined) {
-                throw new Error("session user deleted"); // TODO MFA
+                return {
+                    status: "SESSION_USER_NOT_FOUND_ERROR",
```

Comment:
Could you update the FDI with the new status codes?

Reply:
Also, I think this should be a 401 instead. (as far as I could see it was returned to the frontend)